src/commands/local_storage.ts

```diff
@@ -0,0 +1,19 @@
+import { complement as not, is, isNil } from 'ramda';
+import Message from '../message';
+import { isEmittable } from '../util';
+
+export class Read extends Message {
+  public static expects = { key: is(String), result: isEmittable };
```

Comment:
This looks good to me. My only question is if there is a way to do expectations of a `message` w/o `is` now that we have a type system?

Reply:
This is anticipation of non-TS library consumers.